src/ontology/edits/oeo-physical.omn

```diff
@@ -6071,34 +6072,27 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/911",
     EquivalentTo: 
         (<http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010138)
          and (<http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010140)
-    
-    SubClassOf: 
-        <http://purl.obolibrary.org/obo/BFO_0000015>,
-        OEO_00000532 some OEO_00000006
-    
+
     
 Class: OEO_00010144
 
     Annotations: 
         OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A solid combustion fuel is a combustion fuel that has a solid state of matter under normal conditions.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A solid combustion fuel is a combustion fuel that has a solid state of matter under normal conditions."@en,
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/811
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/931",
         rdfs:label "solid combustion fuel"@en
     
     EquivalentTo: 
         OEO_00000099
          and (OEO_00000529 value OEO_00000390)
-    
-    SubClassOf: 
-        OEO_00000099
```

Comment:
The PR deletes the SubClassOf axioms of the class `solid combustion fuel` (OEO_00010144)